src/interfaces/capnp/proxy.cpp

```diff
@@ -0,0 +1,113 @@
+#include <interfaces/capnp/proxy-impl.h>
+
+#include <kj/async-prelude.h>
+#include <kj/debug.h>
+#include <stdlib.h>
+#include <sys/socket.h>
+#include <unistd.h>
+
+namespace interfaces {
+namespace capnp {
+
+thread_local ThreadContext g_thread_context;
+
+void LoggingErrorHandler::taskFailed(kj::Exception&& exception)
```

Comment:
Not used?

Reply:
re: https://github.com/bitcoin/bitcoin/pull/10102#discussion_r230427653

> Not used?

No, it's a overridden method called through the base class by the kj library.